python/src/trezorlib/debuglink.py

```diff
@@ -487,7 +532,7 @@ def __exit__(self, exc_type, value, traceback):
             # (raises AssertionError on mismatch)
             self._verify_responses(expected_responses, actual_responses)
 
-    def set_expected_responses(self, expected):
+    def set_expected_responses(self, expected: List[MessageFilter]) -> None:
```

Comment:
```
ExpectedMessage = Union[<what from_message_or_type_expects>]
# ... then:
def set_expected_responses(self, expected: Union[ExpectedMessage, Tuple[bool, ExpectedMessage]]])
```

Reply:
Done in b70acaf - hopefully it is fine to use this `ExpectedMessage` as a type of argument into `from_message_or_type`